src/Components/v2/AutoScalingImageLink.tsx

```diff
@@ -0,0 +1,126 @@
+import { color, Flex, Serif, SerifSize } from "@artsy/palette"
+import React from "react"
+import styled from "styled-components"
+import { space, SpaceProps } from "styled-system"
+
+export const AutoScalingImageLink: React.FC<AutoScalingImageLinkProps> = ({
+  href,
+  imageUrl,
+  children,
+  fontSize,
+  maxWidth,
+  maxHeight,
+  ...rest
+}) => {
+  return (
+    <LinkWithBackground href={href} imageUrl={imageUrl} {...rest}>
```

Comment:
Does this link go to a `/collection/id` page?

Reply:
Yes! Good catch on the router stuff. I'll fix.